### PR TITLE
Clean up ios_user SSH tests

### PR DESCRIPTION
**Pull request by KB-perByte**

### CODE REVIEW

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by Qalthos** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

File: tests/integration/targets/ios_user/tests/cli/auth.yaml [hunk 1]
```diff
@@ -72,6 +80,7 @@
       cisco.ios.ios_command:
         commands:
           - show version
+      ignore_errors: true
```
Comment (pabelanger) [inline tests/integration/targets/ios_user/tests/cli/auth.yaml:83]: this is not correct, as we now ignore the failure. We should uses SSH like above

File: tests/integration/targets/ios_user/tests/cli/auth.yaml [hunk 2]
```diff
@@ -12,14 +12,22 @@
...
       meta: reset_connection
 
+    # - name: test login for {{ ansible_user }} user with password
+    #   cisco.ios.ios_command:
+    #     commands:
+    #       - show version
+    #   vars:
+    #     ansible_user: auth_user
+    #     ansible_ssh_pass: pass123
+    #     ansible_private_key_file: ""
```
Comment (pabelanger) [inline tests/integration/targets/ios_user/tests/cli/auth.yaml:22]: is the plan to fix this, if not lets remove

**Comment by pabelanger** [comment]
recheck

File: tests/integration/targets/ios_user/tests/cli/auth.yaml (hunk 1 above)
Comment (KB-perByte) [reply tests/integration/targets/ios_user/tests/cli/auth.yaml:83]: yes I am looking at an alternate way to check this is giving auth_failure

File: tests/integration/targets/ios_user/tests/cli/auth.yaml (hunk 2 above)
Comment (KB-perByte) [reply tests/integration/targets/ios_user/tests/cli/auth.yaml:22]: I add the same test case in a different way, made it same as nxos

File: docs/cisco.ios.ios_logging_global_module.rst [hunk 3]
```diff
@@ -61,7 +61,7 @@ Parameters
                 <td>
                 </td>
                 <td>
-                        <div>Set buffered logging parameters</div>
+                        <div>Set buffered logging parameters.</div>
```
Comment (pabelanger) [inline docs/cisco.ios.ios_logging_global_module.rst:64]: unrelated?

File: plugins/modules/ios_logging_global.py [hunk 4]
```diff
@@ -30,7 +30,7 @@
     type: dict
     suboptions:
       buffered:
-        description: Set buffered logging parameters
+        description: Set buffered logging parameters.
```
Comment (pabelanger) [inline plugins/modules/ios_logging_global.py:33]: unrelated?

File: tests/integration/targets/ios_user/tests/cli/auth.yaml [hunk 5]
```diff
@@ -65,16 +64,17 @@
         state: present
         sshkey: "{{ lookup('file', 'files/test_rsa.pub') }}"
 
-    - name: reset connection with {{ ansible_user }}
+    - name: reset connection with {{ ansible_user }
```
Comment (pabelanger) [inline tests/integration/targets/ios_user/tests/cli/auth.yaml:67]: lets add back the missing brace

File: tests/integration/targets/ios_user/tests/cli/auth.yaml [hunk 6]
```diff
@@ -65,16 +64,17 @@
...
       meta: reset_connection
 
     - name: test sshkey login for {{ ansible_user }} user
       cisco.ios.ios_command:
         commands:
           - show version
+      # ignore_errors: true
```
Comment (pabelanger) [inline tests/integration/targets/ios_user/tests/cli/auth.yaml:74]: lets delete

File: changelogs/fragments/added_entry_galaxy copy.yaml [hunk 7]
```diff
@@ -0,0 +1,3 @@
+---
+trivial:
+  - CheckEE.
```
Comment (pabelanger) [inline changelogs/fragments/added_entry_galaxy copy.yaml:3]: lets expand to clean up ios_user testing for network-ee

**Comment by pabelanger** [comment]
okay, lets remove the depends-on here. Run again against venv, merge. Then we can recheck one last time on EE only jobs and merge

**Comment by pabelanger** [comment]
recheck
